packages/config/config/devices/0x032e/dat-101.json

```
@@ -0,0 +1,402 @@
+{
+	"manufacturer": "DEFARO",
```

```suggestion
	"manufacturer": "Defaro",
```
or is the manufacturer name really written in uppercase?

---

I just copied DAS-102 code

---

Its likely that we didn't clean up that file yet - its a pretty manual process.